### PR TITLE
content on scrum-of-scrums

### LINKED ISSUE #57: content  on scrum-of-scrums
This is for the new proposed outline in #53 

Using the suggested template below as a guide, describe the concept of scrum of scrums . This should include (but not limited to)

- Inputs: What are team members expected to do before scrum of scrums (i.e. determine dependent work or work that may affect other teams)
- Participants
- Outputs: What decisions or actions are needed to conclude the meeting?

### Goals
What are the intended outcomes of the ceremonies?

### Participation
What members of the team should participate?

### Impact
What positive impact has been observed as a result of this practice?

### Measures
How might one measure the impact of this ceremony positive or negative?

### Facilitation Guidance
How might one go about running the ceremony? Is there a general pattern that the ceremony should follow?

### PR DESCRIPTION
**Pull request by meulta**
Resolves #57

### CODE REVIEW

File: scrum-of-scrums/readme.md [hunk 1]
```diff
@@ -1,23 +1,33 @@
 # Scrum of Scrums
 
-] This content is under construction. For now, for those who are unfamiliar: [Scrum of Scrums](https://www.agilealliance.org/glossary/scrum-of-scrums)
+Scrum or scrums is a technique used to scale Scrum to a larger group. In Scrum we consider a team being too big when going over 10-12 individuals. This should be decided on a case by case basis. If the project is set up in multiple workstreams that contain a fixed group of people and a common [stand-up](../stand-ups/readme.md) meeting is slowing down productivity: scrum of scrums should be considered. You would identify the different sub teams that would almost act as separate scrum team.
```
Comment (madossan01) [inline scrum-of-scrums/readme.md:3]: suggestion to complete with: "... to a larger group working under the same project goal." 
Consider also that scrum of scrums may facilitate the communication and help to convey interdependencies on different workstreams.

File: scrum-of-scrums/readme.md [hunk 2]
```diff
@@ -1,23 +1,33 @@
 # Scrum of Scrums
 
-] This content is under construction. For now, for those who are unfamiliar: [Scrum of Scrums](https://www.agilealliance.org/glossary/scrum-of-scrums)
+Scrum or scrums is a technique used to scale Scrum to a larger group. In Scrum we consider a team being too big when going over 10-12 individuals. This should be decided on a case by case basis. If the project is set up in multiple workstreams that contain a fixed group of people and a common [stand-up](../stand-ups/readme.md) meeting is slowing down productivity: scrum of scrums should be considered. You would identify the different sub teams that would almost act as separate scrum team.
 
 ## Goals
 
-What are the intended outcomes of the ceremonies?
+The goal of the scrum of scrums ceremony is to give sub teams the agility they need while not loosing visibility and coordination.
```
Comment (madossan01) [inline scrum-of-scrums/readme.md:7]: An additional and main goal is to guarantee that the sub teams are achieving their sprint goals and they are in the right direction to achieve the overall project goal.

File: scrum-of-scrums/readme.md [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
 ## Goals
 
-What are the intended outcomes of the ceremonies?
+The goal of the scrum of scrums ceremony is to give sub teams the agility they need while not loosing visibility and coordination.
+
+The scrum of scrums ceremony happens every day and can be seen as a regular [stand-up](../stand-ups/readme.md):
+
+    - What was done the day before by the sub team
+    - What will be done today by the sub team 
+    - What are blockers or other issues for the sub team
```
Comment (madossan01) [inline scrum-of-scrums/readme.md:13]: Add " - What are the blockers or issues that may impact other sub teams."

File: scrum-of-scrums/readme.md [hunk 4]
```diff
@@ -1,23 +1,33 @@
...
+This list of impediments is usually managed in a separate [backlog](../backlog-management/readme.md) but does not have to.
 
 ## Participation
 
-What members of the team should participate?
+The common guideline is to have on average one person per sub team to participate in the scrum of scrums. Ideally, the scrum master of each sub team would represent them in this ceremony. In some instances, the representative for the day is selected at the end of each sub team daily [stand-up](../stand-ups/readme.md) and could change every day. In practice, having a fixed representative tends to be more efficient in the long term.
 
 ## Impact
 
-What positive impact has been observed as a result of this practice?
```
Comment (madossan01) [inline scrum-of-scrums/readme.md:15]: List also the drawbacks. For example: some team members will have additional meetings to coordinate and participate. All team members need to be synched later to guarantee a good flow of information etc.

File: scrum-of-scrums/readme.md [hunk 5]
```diff
@@ -1,23 +1,36 @@
 # Scrum of Scrums
 
-] This content is under construction. For now, for those who are unfamiliar: [Scrum of Scrums](https://www.agilealliance.org/glossary/scrum-of-scrums)
+Scrum or scrums is a technique used to scale Scrum to a larger group working under the same project goal. In Scrum we consider a team being too big when going over 10-12 individuals. This should be decided on a case by case basis. If the project is set up in multiple workstreams that contain a fixed group of people and a common [stand-up](../stand-ups/readme.md) meeting is slowing down productivity: scrum of scrums should be considered. You would identify the different sub teams that would almost act as separate scrum team.
```
Comment (bsherwin) [inline scrum-of-scrums/readme.md:3]: Second word "or" should be "of"

Comment (bsherwin) [inline scrum-of-scrums/readme.md:3]: "working under" may be better to say "working toward the same project goal"

Comment (bsherwin) [inline scrum-of-scrums/readme.md:3]: "You would identify the different sub teams that would almost act as separate scrum team."

Consider: "The team would identify the different sub teams that would act as a separate scrum teams with their own backlog, board and standup."

Comment (meulta) [reply scrum-of-scrums/readme.md:3]: Thanks!

**Comment by meulta** [comment]
@bsherwin thanks for the comments, I just made an update :)

File: scrum-of-scrums/readme.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 # Scrum of Scrums
 
-Scrum or scrums is a technique used to scale Scrum to a larger group working under the same project goal. In Scrum we consider a team being too big when going over 10-12 individuals. This should be decided on a case by case basis. If the project is set up in multiple workstreams that contain a fixed group of people and a common [stand-up](../stand-ups/readme.md) meeting is slowing down productivity: scrum of scrums should be considered. You would identify the different sub teams that would almost act as separate scrum team.
+Scrum of scrums is a technique used to scale Scrum to a larger group working towards the same project goal. In Scrum we consider a team being too big when going over 10-12 individuals. This should be decided on a case by case basis. If the project is set up in multiple workstreams that contain a fixed group of people and a common [stand-up](../stand-ups/readme.md) meeting is slowing down productivity: scrum of scrums should be considered. The team would identify the different sub groups that would act as a separate scrum teams with their own backlog, board and [stand-up](/scrum-of-scrums/readme.md).
```
Comment (bsherwin) [inline scrum-of-scrums/readme.md:3]: I think the link on standup is pointing to the wrong place.

Comment (meulta) [reply scrum-of-scrums/readme.md]: fixed

**Pull request merged**